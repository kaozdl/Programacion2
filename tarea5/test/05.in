# Caso 05
procesar_rango 2
reiniciar_sistema
ingresar_localidades (50,tambo) .
procesar_rango 2
ingresar_localidades (34,ulsan) (4,goya) (23,esmeraldas) (65,cumana) .
procesar_rango 2
procesar_rango 1
procesar_rango 5
procesar_rango 2
Fin
