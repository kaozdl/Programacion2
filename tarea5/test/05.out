1>2>3>
  REINICIO DE SISTEMA 
4>5>6>7>esmeraldas

ulsan

8>goya

9>10>esmeraldas

ulsan

11>Fin.
